src/include/daos_types.h

```diff
@@ -139,10 +144,21 @@ typedef enum {
 typedef struct {
 	uint16_t	da_type; /** daos_anchor_type_t */
 	uint16_t	da_shard;
-	uint32_t	da_padding;
+	uint32_t	da_flags;
```

Comment:
comment please, otherwise it's hard for user to understand use-case of this flag

Reply:
OK